python/lsst/pipe/base/argumentParser.py

```diff
@@ -564,8 +564,10 @@ def __init__(self, pattern):
 
                     def write(self, str):
                         str = str.rstrip()
-                        if str and fnmatch.fnmatch(str, self._pattern):
-                            print str
+                        # Strip off doc string line(s) and cut off at "=" for string matching
+                        matchStr = str.split("\n")[-1].split("=")[0]
```

Comment:
You could make this even shorter by including the `rstrip` directly -- `matchStr = str.rstrip().split(...)`, etc. Whether that's gaining or losing you readability is arguable, I suppose.